src/components/mainmenu/MainMenu.tsx

```diff
@@ -54,11 +57,11 @@ export const MainMenu: FC<MainMenuProps> = ({ tabs }) => {
                 tabs.current?.close();
             },
             catchEmAll: () => {
-                openFursuitGames().catch(captureException);
+                openFursuitGames(t).catch(captureException);
                 tabs.current?.close();
             },
             services: () => {
-                openAdditionalServices().catch(captureException);
+                openAdditionalServices(t).catch(captureException);
```

Comment:
Please add `t` to the dependency list at the end of the hook (`[t, tabs, openFursuitGames, openAdditionalServices, login],`)

The hook needs to know when to recompute, even tho `t` is unlikely to change it is bad practice not to include it.